packaging/conda/build.sh

```diff
@@ -91,8 +80,8 @@ then
     sudo security unlock-keychain -p "$KHIOPS_APPLE_TMP_KEYCHAIN_PASSWORD" kh-tmp.keychain
 
     # Add identity (certificate + private key) to the temporary keychain
-    echo "$KHIOPS_APPLE_CERTIFICATE_BASE64" \
-      | base64 --decode -i - -o kh-cert.p12
+    echo "$KHIOPS_APPLE_CERTIFICATE_BASE64" |
+      base64 --decode -i - -o kh-cert.p12
```

Comment:
Do we need to have these on two separate lines?

Reply:
no...